src/token.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Example:
+ * \{
+ * 	 id: 1,
+ *   symbol: "WETH",
+ *   decimals: 18,
+ *   address: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
```

Comment:
nit: I would prefer these to be included in documentation for the individual parameters in `TokenInfo`.